autorest/codegen/templates/operation.py.jinja2

```diff
@@ -36,7 +36,7 @@
 {% endif %}
 
     # Construct URL
-    url = self.{{ operation.python_name }}.metadata['url']
+    url = self.{{ operation.python_name }}.metadata['url']  # type: ignore
```

Comment:
what about `# type: str` ?

Reply:
The typing error I'm trying to ignore is ""Callable[..., Awaitable[str]]" has no attribute "metadata"" (it really doesn't like the setting of the metadata out of the operation), so changing it to # type: str doesn't solve that issue

Reply:
gotcha